security/access_token.rst

```diff
@@ -343,6 +343,438 @@ and configure the service ID as the ``success_handler``:
     ``failure_handler`` option and create a class that implements
     :class:`Symfony\\Component\\Security\\Http\\Authentication\\AuthenticationFailureHandlerInterface`.
 
+Using OIDC
```

Comment:
I think using the full protocol name (OpenID Connect) would be good here, either in parenthesis or vice versa